src/logger/logger.ts

```diff
@@ -0,0 +1,20 @@
+import {createLogger, format, transports} from 'winston';
+
+const {combine, timestamp, printf} = format;
+
+// tslint:disable-next-line:no-shadowed-variable
```

Comment:
its really confusing 2 variables with the same name in the same module - don't do this

Reply:
fixed it